src/pages/2020_2021.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react";
+import { chakra, Box, Flex, useColorModeValue, Link } from "@chakra-ui/react";
+import vvcoding from "../images/vvvcoding.svg";
+import EventHeader from "../components/EventHeader";
+import { Stack, HStack, VStack } from "@chakra-ui/react";
+
+class CurrentYear extends Component {
+  render() {
+    return (
+      <Flex p={30} w="full" alignItems="center" justifyContent="left">
+        <VStack spacing={400} align="stretch">
+          <Box>
+            <EventHeader />
+          </Box>
+        </VStack>
+
+        <Box
+          as="span"
+          //  bg={useColorModeValue("white", "gray.900")}
+          // mx={{ md: 3 }}
+          display="flex"
+          width="800px"
+          height="305px"
+          borderWidth="2px"
+          borderRadius="lg"
+        >
+          <Box w={{ lg: "45%" }}>
+            <Box
+              h={{ base: 14, sm: "full" }}
+              rounded={{ lg: "lg" }}
+              bgSize="cover"
+              bgImage={vvcoding}
+              width={300}
+              bgSize="contain"
```

Comment:
Duplicate prop is present here.
Are you using eslint ? As I have mentioned in [readme](https://github.com/sam9111/ssn-acm-w/blob/master/README.md)

Reply:
hey this page is not used.My mistake for commiting it.